aptos-move/e2e-move-tests/src/tests/attributes.rs

```diff
@@ -30,6 +30,66 @@ fn test_view_attribute() {
     assert_success!(h.publish_package(&account, path.path()));
 }
 
+#[test]
+#[should_panic]
+fn test_view_attribute_with_signer() {
+    let mut h = MoveHarness::new();
+    let account = h.new_account_at(AccountAddress::from_hex_literal("0xf00d").unwrap());
+
+    let mut builder = PackageBuilder::new("Package");
+    builder.add_source(
+        "m.move",
+        r#"
+        module 0xf00d::M {
+            #[view]
+            fun view(_:signer,value: u64): u64 { value }
+        }
+        "#,
+    );
+    let path = builder.write_to_temp().unwrap();
+    assert_success!(h.publish_package(&account, path.path()));
```

Comment:
Can you add a comment saying that we are panic in the publish request?

Reply:
> Can you add a comment saying that we are panic in the publish request?

This should not be published because an error will be reported during compilation, so I need to delete the publish assertion here.